### ISSUE TITLE
Codesigning in "[CP] Embed Pods Frameworks" script fails if app has space in name

### ISSUE THREAD

**Issue by boisy** [CocoaPods/CocoaPods#6354]
> Before you start, are you using the latest CocoaPods release?
> A lot changes with Xcode releases that are not backwards compatible.

pod --version shows 1.2.0.beta.1

> Xcode 8 Status: Only works with CocoaPods 1.1.0+
> Pushing a Swift 2.3 Pod: You need to add a `.swift-version` file or the argument `--swift-version=VERSION`, see the CHANGELOG
> Not an issue about the CocoaPods command line app? Please file an issue in the appropriate repo - https://github.com/CocoaPods
> Issues are for feature requests, and bugs; questions should go to Stack Overflow
> Using CocoaPods <= 0.39: http://blog.cocoapods.org/Sharding/

* [x ] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?
pod install, then built my project

## What did you expect to happen?
Project to be built without error

## What happened instead?
Project failed at [CP] Embed Pods Frameworks script

## CocoaPods Environment
[in /Users/boisy/Projects/WeatherSnoop4]

### Stack

```
   CocoaPods : 1.2.0.beta.1
        Ruby : ruby 2.0.0p648 (2015-12-16 revision 53162) [universal.x86_64-darwin16]
    RubyGems : 2.0.14.1
        Host : Mac OS X 10.12.2 (16C67)
       Xcode : 8.2.1 (8C1002)
         Git : git version 2.10.1 (Apple Git-78)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 4c3e52366283b3e0199826858a42c174b0bc0d43
               twilio - https://github.com/twilio/cocoapod-specs.git @ 7162facbbcc3634678b08af02be19cf2ce1bf676
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.1.2
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
source 'https://github.com/CocoaPods/Specs.git'

platform :osx, '10.11'
workspace 'WeatherSnoop4.xcworkspace'

target 'WeatherSnoop 4' do  
	pod 'TeeBoy'
	pod 'Sparkle'
	pod 'Paddle'
end

target 'WS4Kit' do  
	pod 'PXSourceList', '~> 2.0'
	pod 'CocoaAsyncSocket', '~> 7.5'
	pod 'AFNetworking', '~> 3.0'
	pod 'CorePlot'
	pod 'GCDWebServer'
	pod 'Paddle'
	pod 'TeeBoy'
end

target 'WS4Kit Tests' do  
	pod 'TeeBoy'
end

# Agent Plug-Ins
target 'ColumbiaWeatherSystemsMicroServerAgent' do  
	project 'plugins/Agent PlugIns/ColumbiaWeatherSystemsMicroServerAgent/ColumbiaWeatherSystemsMicroServerAgent'
	pod 'TeeBoy'
end

target 'DavisAgent' do  
	project 'plugins/Agent PlugIns/DavisAgent/DavisAgent'
	pod 'TeeBoy'
end

target 'PeetBrosAgent' do  
	project 'plugins/Agent PlugIns/PeetBrosAgent/PeetBrosAgent'
	pod 'TeeBoy'
end

target 'RainWiseCC3000Agent' do  
	project 'plugins/Agent PlugIns/RainWiseCC3000Agent/RainWiseCC3000Agent'
	pod 'TeeBoy'
end

target 'FineOffsetAgent' do  
	project 'plugins/Agent PlugIns/FineOffsetAgent/FineOffsetAgent'
	pod 'TeeBoy'
end

target 'WeatherUndergroundAgent' do  
	project 'plugins/Agent PlugIns/WeatherUndergroundAgent/WeatherUndergroundAgent'
	pod 'TeeBoy'
end

target 'TexasWeatherInstrumentsAgent' do  
	project 'plugins/Agent PlugIns/TexasWeatherInstrumentsAgent/TexasWeatherInstrumentsAgent'
	pod 'TeeBoy'
end

target 'WeatherHawkAgent' do
        project 'plugins/Agent PlugIns/WeatherHawkAgent/WeatherHawkAgent'
        pod 'TeeBoy'
end

# Site Plug-Ins
target 'LocationPlugIn' do
	project 'plugins/Site PlugIns/LocationPlugIn/LocationPlugIn'
	pod 'TeeBoy'
end

target 'PropertiesPlugIn' do
	project 'plugins/Site PlugIns/PropertiesPlugIn/PropertiesPlugIn'
	pod 'TeeBoy'
end


# Sharing Plug-Ins
target 'WeatherUndergroundSharingPlugIn' do
	project 'plugins/Sharing PlugIns/WeatherUndergroundSharingPlugIn/WeatherUndergroundSharingPlugIn'
	pod 'TeeBoy'
end

target 'WeathercloudSharingPlugIn' do
	project 'plugins/Sharing PlugIns/WeathercloudSharingPlugIn/WeathercloudSharingPlugIn'
	pod 'TeeBoy'
end

target 'PWSSharingPlugIn' do
	project 'plugins/Sharing PlugIns/PWSSharingPlugIn/PWSSharingPlugIn'
	pod 'TeeBoy'
end

target 'MetOfficeWOWSharingPlugIn' do
	project 'plugins/Sharing PlugIns/MetOfficeWOWSharingPlugIn/MetOfficeWOWSharingPlugIn'
	pod 'TeeBoy'
end

target 'CWOPSharingPlugIn' do
	project 'plugins/Sharing PlugIns/CWOPSharingPlugIn/CWOPSharingPlugIn'
	pod 'TeeBoy'
end

target 'WeatherBugSharingPlugIn' do
	project 'plugins/Sharing PlugIns/WeatherBugSharingPlugIn/WeatherBugSharingPlugIn'
	pod 'TeeBoy'
end

target 'DashboardPlugIn' do
	project 'plugins/Misc PlugIns/DashboardPlugIn/DashboardPlugIn'
	pod 'TeeBoy'
	pod 'Paddle'
end

# Misc Plug-Ins
target 'ReportsPlugIn' do
	project 'plugins/Report PlugIns/ReportsPlugIn/ReportsPlugIn'
	pod 'TeeBoy'
	pod 'Paddle'
end
```

## Project that demonstrates the issue

I'm not submitting the project. Instead, I'll show you what I did to fix the issue.

What I had to do was edit the resulting Pods/Target Support Files/Pods-WeatherSnoop 4/Pods-WeatherSnoop 4-frameworks.sh

    local code_sign_cmd="/usr/bin/codesign --force --sign ${EXPANDED_CODE_SIGN_IDENTITY} ${OTHER_CODE_SIGN_FLAGS} --preserve-metadata=identifier,entitlements "$1""

to

    local code_sign_cmd="/usr/bin/codesign --force --sign ${EXPANDED_CODE_SIGN_IDENTITY} ${OTHER_CODE_SIGN_FLAGS} --preserve-metadata=identifier,entitlements "\"$1\"""

To conclude, the CocoaPods entity responsible for creating this script is not taking into account target names with spaces in them.

**Comment by samuelbeek** [comment]
I had the same problem with an application that had spaces (the target did not). The solution that @boisy proposed works for me as well.

**Comment by dnkoutso** [comment]
This should be fixed with 1.2.0.beta.3 I think, can you please give this a shot?

**Comment by samuelbeek** [comment]
@dnkoutso that fixed it for me, thanks 💯

**Comment by dnkoutso** [comment]
nice1

@boisy can you confirm as well? If so, please close this issue as well!

**Comment by boisy** [comment]
Hi Dimitrus,

Just tried it and I can confirm it’s fixed! Great work!

I’ll close the issue.

--
Boisy G. Pitre / Lead Developer
 <http://www.tee-boy.com/>support@tee-boy.com <mailto:boisy@tee-boy.com>
Tee-Boy 
441 Saint Paul Avenue • Opelousas, LA 70570
http://www.tee-boy.com <http://www.tee-boy.com/>
This e-mail message may contain confidential or legally privileged information and is intended only for the use of the intended recipient(s). Any unauthorized disclosure, dissemination, distribution, copying or the taking of any action in reliance on the information herein is prohibited. E-mails are not secure and cannot be guaranteed to be error free as they can be intercepted, amended, or contain viruses. Anyone who communicates with us by e-mail is deemed to have accepted these risks. Company Name is not responsible for errors or omissions in this message and denies any responsibility for any damage arising from the use of e-mail. Any opinion and other statement contained in this message and any attachment are solely those of the author and do not necessarily represent those of the company. <http://htmlsig.com/www.linkedin.com/boisypitre>

**Comment by boisy** [comment]
Fixed in 1.2.0.beta.3. Thanks!

**Comment by SalmanAhmedDev** [comment]
i have just update the pod version : sudo gem install cocoapods work for me.
